CHANGELOG.rst

```diff
@@ -0,0 +1,49 @@
+OMEGO version history
+=====================
+
+0.3.0 (February 2016)
+---------------------
+
+ * drops support for OMERO 5.1 and earlier
```

Comment:
Installation of 5.0 still works, it's just that some options like prefixing ports isn't possible anymore.